cloverage/test/cloverage/coverage_test.clj

```diff
@@ -378,7 +389,9 @@
        "-x" "cloverage.sample.exercise-instrumentation"
        "cloverage.sample.exercise-instrumentation")
       (doseq [fname generated-files]
-        (t/is (equal-content? fname "out" "test/resources") (str "Failing for file: " fname))))))
+        ;; If this deftest is failing, you can temporarily enable this to update the expectations:
+        #_(spit (str "test/resources/" fname) (slurp (str "out/" fname)))
+        (assert-equal-content! fname "test/resources" "out")))))
```

Comment:
Swapped `"test/resources" "out"` order so that it resembles "expected / actual" which is a standard (the Kaocha diffs will also make more sense)